h2/src/main/org/h2/server/pg/PgServerThread.java

```diff
@@ -926,7 +926,12 @@ private void initDb() throws SQLException {
             synchronized (server) {
                 // better would be: set the database to exclusive mode
                 boolean tableFound;
-                try (ResultSet rs = conn.getMetaData().getTables(null, "PG_CATALOG", "PG_VERSION", null)) {
+                String pgCatalog = "PG_CATALOG", pgVersion = "PG_VERSION";
+                if (((JdbcConnection) conn).getSession().getStaticSettings().databaseToLower) {
+                    pgCatalog = pgCatalog.toLowerCase();
+                    pgVersion = pgVersion.toLowerCase();
```

Comment:
Rather call StringUtils.toLowerEnglish otherwise this might do weird stuff in other locales.

Reply:
ok. just follows `database.sysIdentifier()`.